packages/amplify-codegen-e2e-tests/src/__tests__/build-app-swift.test.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  initProjectWithQuickstart,
+  DEFAULT_IOS_CONFIG,
+  addApiWithBlankSchemaAndConflictDetection,
+  updateApiSchemaWithText,
+  generateModels,
+  swiftBuild,
+} from '@aws-amplify/amplify-codegen-e2e-core';
+const { schemas } = require('@aws-amplify/graphql-schema-test-library');
+import { existsSync, writeFileSync, readdirSync, rmSync, readFileSync } from 'fs';
+import path from 'path';
+
+const skip = new Set([
```

Comment:
These cases are consistently failing. This indicates an existing issue with codegen for these cases.